sunpy/coordinates/frames.py

```diff
@@ -169,17 +173,26 @@ class BaseHeliographic(SunPyBaseCoordinateFrame):
                                 RepresentationMapping('d_distance', 'd_radius', u.km/u.s)],
     }
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    rsun = QuantityAttribute(default=_RSUN, unit=u.km)
 
-        self._make_3d()
+    def make_3d(self):
+        """
+        This method calculates the third coordinate of the frame.
```

Comment:
```suggestion
        This method calculates the third (radial) coordinate of the frame.
```
If it's always the radial coordinate that's added with this method?

Reply:
Greatly expanded the description